### PR TITLE
Enable docker on marathon containerizers.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: templates/executor_registration_timeout [hunk 1]
```diff
@@ -0,0 +1 @@
+5mins
```
Comment (akatsoulas) [inline templates/executor_registration_timeout:1]: What about moving this into a variable?

**Comment by johngian** [comment]
@akatsoulas 
PR updated with the following changes:
- Add required docker setup in slave nodes
- Move all hardcoded values to vars

**Comment by johngian** [comment]
@akatsoulas Tested it locally and launched "libmesos/ubuntu" with test configuration [1] succesfully.

[1] https://gist.github.com/johngian/fb9cb98dcbacddab766f

**Comment by akatsoulas** [comment]
r+

**Comment by johngian** [comment]
Tested it locally and seems to be ok.

**Pull request merged**